test/chplcheck/MethodsAfterFields.chpl

```diff
@@ -1,53 +1,54 @@
-record emptyRecord {   
-}
-
-record methodField {
-    proc method1() {
-        return 12;
+module MethodsAfterFields {
+    record emptyRecord {   }
```

Comment:
Here and elsewhere, please use consistent indentation. Chapel style is to use 2-space indentation.